inc/views/base/newthread/newthread.twig

```diff
@@ -0,0 +1,227 @@
+{% extends 'layouts/master.twig' %}
+
+{% block head %}
+    <title>{{ trans('newthread_in', forum.name) }}</title>
+    <script type="text/javascript" src="{{ asset_url('jscripts/post.js') }}"></script>
+{% endblock head %}
+
+{% block body %}
+    {% if newthread.preview %}
+        <table border="0" cellspacing="{{ theme.borderwidth }}" cellpadding="{{ theme.tablespace }}" class="tborder tfixed clear no_bottom_border">
+            <tr>
+                <td class="thead"><strong>{{ lang.post_preview }}</strong></td>
+            </tr>
+            <tr>
+                <td id="posts_container">
+                    <div id="posts">
+                        {{ postbit|raw }}
+                    </div>
+                </td>
+            </tr>
+        </table>
+        <br />
+    {% endif %}
+    {{ thread_errors|raw }}
+    {% if newthread.showmodnotice %}
+        <div class="pm_alert">
+            {{ newthread.moderation_text }}
+        </div>
+    {% endif %}
+    <form action="newthread.php?fid={{ forum.fid }}&amp;processed=1" method="post" enctype="multipart/form-data" name="input">
+        <input type="hidden" name="my_post_key" value="{{ mybb.post_code }}" />
+        <table border="0" cellspacing="{{ theme.borderwidth }}" cellpadding="{{ theme.tablespace }}" class="tborder">
+            <tr>
+                <td class="thead" colspan="2"><strong>{{ lang.post_new_thread }}</strong></td>
+            </tr>
+            {{ loginbox|raw }}
+            <tr>
+                <td class="trow2" width="20%"><strong>{{ lang.thread_subject }}</strong></td>
```

Comment:
As in the other PR, would you mind adding `<label>` elements where we have raw text labels for inputs? Makes them harder to miss for the new theme as it gets written that way 😄